pkgs/applications/networking/browsers/luakit/default.nix

```
@@ -27,7 +27,8 @@ in stdenv.mkDerivation rec {
     gst_all_1.gstreamer gst_all_1.gst-plugins-base
     gst_all_1.gst-plugins-good gst_all_1.gst-plugins-bad gst_all_1.gst-plugins-ugly
     gst_all_1.gst-libav
-                ];
+    glib-networking
```

Can you add
```suggestion
    glib-networking # TLS support
```

So we can be clear on why it's there?